src/main/java/frc/robot/lib/interfaces/Swerve.java

```diff
@@ -342,8 +352,8 @@ public double setClosestPose(Pose2d currentPose) {
         // get distance between current pose and target pose of every target pose 
```

Comment:
So this is the function that sets the closest pose from ALL our defined poses. So, it should be checking ALL the poses, including both the substation poses. At the moment it only checks ONE of the substation poses, the one at 0 position in the array.
We will also need to re-think what the `poseNumber` variable means in the context of having two loading station poses. Right now, we just assumed that if it's at the substation there will only ever be one pose, so we made it -1. But, with 2 poses there, now we can theoretically allow the driver to shift between the two using the bumpers on the controller. The bumper press by the driver calls the `shiftPose()` function, which does nothing when the `poseNumber` is -1. This would have to be re-tooled to allow shifting at the substation. Do we even want shifting at the substation? Food for thought.
In any case, we need to check all the substation poses in this function in the same manner we check the scoring poses for this function to work.